messages/preview.json

```diff
@@ -3,8 +3,12 @@
     "platformFlagDescription": "Specify platform ('iOS' or 'Android').",
     "targetFlagDescription": "Specify name of target simulator or emulator.",
     "targetAppFlagDescription": "Identifies the target app. Defaults to the browser app.",
-    "targetAppArgumentsFlagDescription": "Comma separated values defining extra arguments to be passed to the target app.",
+    "configHelpFlagDescription": "Displays the format of the preview config file.",
+    "configFileFlagDescription": "Extended configuration options for preview. Note: this is required for previewing against a mobile app, as apps require extended configuration.",
     "componentnameFlagDescription": "Component name.",
     "projectDirFlagDescription": "Path to the root of the component project. Defaults to the current working directory.",
-    "error:invalidInputFlagsDescription": "Invalid input values for flags."
+    "ignoringConfigFileFlagDescription": "Configuration file will be ignored when target app is browser.",
+    "error:invalidInputFlagsDescription": "Invalid input values for flags.",
+    "error:invalidConfigFile:genericDescription": "Invalid configuration file: %s",
```

Comment:
A couple of things:

1. This message is slightly vague. Let's give it verbiage like "Configuration file has an invalid format: %s"
2. It would also be good to include the full path to the configuration file as part of the error message. It lets the consumer spot-validate the config file path and make sure they didn't fat-finger something.